database/seed.js

```diff
@@ -1,55 +1,227 @@
-const Question = require('./question.js');
-const db = require('./index.js');
+const db = require('./index');
 const faker = require('faker');
-const moment = require('moment');
-
-let answers = [], data = [];
-let keywords = [
-  'PS5', 'XBOX', 'controller', 'steam', 'playstation', 'console', 'games', 'joystick', 'configuration',
-  'order', 'money', 'system', 'expensive', 'cheap', 'wireless', 'connection', 'light', 'battery', 'life', 'compatible', 'video', 'please help me'
-];
-
-for (var i = 0; i < 10000; i++) {
-  let word = keywords[faker.random.number({min: 0, max: keywords.length -1})];
-  let para = faker.lorem.paragraph(faker.random.number({min: 1, max: 10}));
-  if (Math.random() > 0.99) {
-    let sentence = para.split('.');
-    let random = faker.random.number({min: 0, max: sentence.length - 1});
-    let words = sentence[random].split(' ');
-    let rand = faker.random.number({min: 1, max: words.length - 1});
-    words[rand] = word;
-    sentence[random] = words.join(' ');
-    para = sentence.join('.');
+const Sequelize = require('sequelize');
+const { createObjectCsvWriter } = require('csv-writer');
+const { performance } = require('perf_hooks');
+const {
+  Items,
+  Users,
+  Questions,
+  Answers
+} = require('./Model');
+
+// const Promise = require('bluebird');
+// const { Pool } = require('pg');
+
+// let pool = new Pool({
+//   host: 'localhost',
+//   user: 'postgres',
+//   password: 'postgres',
+// });
+
+// pool = Promise.promisifyAll(pool);
+
+// pool.connectAsync()
+//   .catch((err) => { console.error(err); });
+
+const seed = async () => {
+
+  let start = performance.now();
+
+  await Answers.drop();
+  await Questions.drop();
+  await Users.drop();
```

Comment:
this should be a loop - 

`await Promise.all([obj1, obj2 ...].map(async obj => {await obj.drop(); await obj.sync()}))`